api/src/main/java/com/datastrato/gravitino/authorization/Privilege.java

```diff
@@ -21,7 +21,67 @@ public interface Privilege {
 
   /** The name of this privilege. */
   enum Name {
-    /** The privilege of load a catalog. */
-    LOAD_CATALOG
+    /** The privilege to list catalogs. */
+    LIST_CATALOG,
+    /** The privilege to load a catalog. */
+    LOAD_CATALOG,
+    /** The privilege to create a catalog. */
+    CREATE_CATALOG,
+    /** The privilege to alter a catalog. */
+    ALTER_CATALOG,
+    /** The privilege to drop a catalog. */
+    DROP_CATALOG,
+    /** The privilege to list schemas. */
+    LIST_SCHEMA,
+    /** The privilege to load a schema. */
+    LOAD_SCHEMA,
+    /** The privilege to create a schema. */
+    CREATE_SCHEMA,
+    /** The privilege to alter a schema. */
+    ALTER_SCHEMA,
+    /** The privilege to drop a schema. */
+    DROP_SCHEMA,
+    /** The privilege to list tables. */
+    LIST_TABLE,
+    /** The privilege to load a table. */
+    LOAD_TABLE,
+    /** The privilege to alter a table. */
+    ALTER_TABLE,
+    /** The privilege to create a table. */
+    CREATE_TABLE,
+    /** The privilege to drop a table. */
+    DROP_TABLE,
+    /** The privilege to read a table. */
+    READ_TABLE,
```

Comment:
Do we need both LOAD_XXX and READ/WRITE_XXX? How will user use them?

Reply:
Also how do you distinguish whether the table has read or write privilege from Gravitino?

Reply:
1. Yes, LOAD_TABLE is a meta data operation and the READ_TABLE is a data operation. They are different operation.
2. Gravitino doesn't disitinguish the privilege. User doesn't use Gravitino to read or write table.
Gravitino set the privileges of read/ write table to the underlying system. The underlying system will judge the privilege of read/write when the users access the underlying system.